spoon-runner/src/main/resources/page/log.html

```diff
@@ -17,6 +17,7 @@
             <div class="hero-unit">
                 <h1>{{title}}</h1>
                 <p>{{subtitle}}</p>
+                <a href="{{rawFile}}">Raw log file</a>
```

Comment:
Do you have a screenshot of what this looks like? I'm not sure this is the right place for the link.

Reply:
yeah i wasn't super happy with it.  maybe put a different link on the main page next to the html log link?